kategory/src/main/kotlin/kategory/data/Const.kt

```diff
@@ -0,0 +1,39 @@
+package kategory
+
+fun <A, T> ConstKind<A, T>.value(): A = this.ev().value
```

Comment:
We decided not to do it because it required creating lots of code overhead that would bloat the library.